### DISCUSSION TITLE
Error running the *.test site

### DISCUSSION THREAD

**Discussion by iRabiN19** [beyondcode/herd-community#550] [Q&A]
I had set up herd as the installation said and tried accessing the project at <folder-name>.test but the browser couldn't find the page.
After reopening herd, I found the herdhelper port status to be red and went through the logs to find the error even after granting the admin privilege:
An error occurred: Access to the path 'C:\Windows\System32\drivers\etc\hosts' is denied.

**Comment by GL-MorganC** [comment]
In fact, Herd try to write some DNS resolutions in your  "C:\Windows\System32\drivers\etc\hosts" file.
```
127.0.0.1   yoursite.test
127.0.0.1   yoursecond.text
```

On the property of your host file, uncheck "read only" and try to reboot Herd (try to run it as an administrator too). It should write some information on it
